sdk/identity/azure-identity/README.md

```diff
@@ -84,6 +86,11 @@ You may notice in `InteractiveBrowserCredentialBuilder`, a port number is requir
 
 In this case, the port number is 8765.
 
+#### Enable applications for shared token cache credential
+You will need to have Visual Studio 2019 installed. Login to Visual Studio with your org ID or live ID and you are ready to use shared token cache credential.
+
+If you see an error "MSAL V3 Deserialization failed", try clearing the cache in `C:\Users\{username}\AppData\.IdentityService`.
```

Comment:
Is the error actually "MSAL V3 Deserialization failed to parse the cache contents."?

Reply:
![image](https://user-images.githubusercontent.com/2320445/64659737-22266a00-d3f2-11e9-9645-0dbc8f9d2cc9.png)